### DISCUSSION TITLE
Working Directory of Quick Terminal

### DISCUSSION THREAD

**Discussion by nihklas** [ghostty-org/ghostty#3726] [Feature Requests, Ideas] [answered]
I love the fact that the working directory gets inherited in new Terminal Windows and Tabs. But the Quick Terminal doesn't really fit this use case. If i need a short lived terminal, when im already inside the terminal i just open another tab and close it when im done. I use the quick terminal only when im using other apps in the meantime. It still inherits the working directory of the currently still opened ghostty session. 

I would like to set a separate working directory option specifically for the quick terminal.

**Comment by JakeWharton** [comment, marked as answer]
Probably can be added to #1392
